deploy/docker/schemas.py

```
@@ -4,11 +4,15 @@
 from utils import FilterType
 
 
-class CrawlRequest(BaseModel):
-    urls: List[str] = Field(min_length=1, max_length=100)
+
+class CrawlConfigs(BaseModel):
     browser_config: Optional[Dict] = Field(default_factory=dict)
     crawler_config: Optional[Dict] = Field(default_factory=dict)
 
+class CrawlRequest(BaseModel):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)